presto-hive/src/main/java/com/facebook/presto/hive/HiveWritableTableHandle.java

```diff
@@ -120,18 +121,33 @@ public LocationHandle getLocationHandle()
         return preferredOrderingColumns;
     }
 
+    /* This field is deprecated and clients should switch to {@link #getActualStorageFormat()}*/
```

Comment:
nit: perhaps, shorten to `Use {@link #getActualStorageFormat()};` @deprecated annotation already says "This field is deprecated"